<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>1250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>1250</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1605</width>
    <height>900</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Packet Sniffer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Monospace</family>
    </font>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>1302</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1300</width>
      <height>600</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>1300</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#tableWidget::item {
    background-color: #ffffff;
    color: #000000;
    border: transparent;
    border-bottom: 1px solid #dbdbdb;
    padding: 8px;
}


#tableWidget {
	border: 2px solid;
}</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>包序号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>源MAC</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的MAC</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>源ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>源端口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的端口</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>800</y>
      <width>1302</width>
      <height>400</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #000000;</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="ruleButton1">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid #000000;
}

QPushButton:hover {
	background-color: rgb(200, 200, 200);
}

QPushButton:pressed {
	background-color: rgb(230, 230, 230);
	padding-left:3px;
    padding-top:3px;
}</string>
     </property>
     <property name="text">
      <string>rule1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ruleButton2">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid #000000;
}

QPushButton:hover {
	background-color: rgb(200, 200, 200);
}

QPushButton:pressed {
	background-color: rgb(230, 230, 230);
	padding-left:3px;
    padding-top:3px;
}</string>
     </property>
     <property name="text">
      <string>rule2</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ruleButton3">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid #000000;
}

QPushButton:hover {
	background-color: rgb(200, 200, 200);
}

QPushButton:pressed {
	background-color: rgb(230, 230, 230);
	padding-left:3px;
    padding-top:3px;
}</string>
     </property>
     <property name="text">
      <string>rule3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid #000000;
	color: rgb(255, 255, 255);
	background-color: rgb(44, 45, 46);
}

QPushButton:hover {
	background-color: rgb(80, 80, 80);
}

QPushButton:pressed {
	background-color: rgb(60, 60, 60);
	padding-left:3px;
    padding-top:3px;
}</string>
     </property>
     <property name="text">
      <string>抓包</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="pnum_label">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #000000;</string>
    </property>
    <property name="text">
     <string>抓包数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pnum_input">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #000000;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="promisc_label">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>30</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #000000;</string>
    </property>
    <property name="text">
     <string>混杂模式</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="promisc_flag">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>30</y>
      <width>100</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #000000;</string>
    </property>
    <item>
     <property name="text">
      <string>Off</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>On</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>90</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 2px solid #000000;
}

QPushButton:hover {
	background-color: rgb(200, 200, 200);
}

QPushButton:pressed {
	background-color: rgb(230, 230, 230);
	padding-left:3px;
    padding-top:3px;
}</string>
    </property>
    <property name="text">
     <string>追踪进程流</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>90</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 2px solid #000000;
}

QPushButton:hover {
	background-color: rgb(200, 200, 200);
}

QPushButton:pressed {
	background-color: rgb(230, 230, 230);
	padding-left:3px;
    padding-top:3px;
}</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="actionzajina_1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>zajina[1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
